circle.yml

```diff
@@ -0,0 +1,13 @@
+machine:
+  node:
+    version: 6  # this specific version is preinstalled on the CI servers and allows faster builds
+test:
```

Comment:
Is this block necessary ?
Without it, circle will probably just run `npm test`. 
If you remove your custom `npm test`, it should display a fine message and not raise any error.